l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py

```diff
@@ -24,7 +37,7 @@ class XafAuditfileExport(models.Model):
 
     @api.depends('name')
     def _auditfile_name_get(self):
-        self.auditfile_name = '%s.xaf' % self.name
+        self.auditfile_name = '%s.xaf.zip' % self.name
```

Comment:
May I suggest to make field `auditfile_name` as stored? We need to preserve the original name in the old generated auditfiles.

Moreover, the calculation of the file name could be deducted from the file type. What I mean is this: 
https://github.com/onesteinbv/l10n-netherlands/blob/11_imp_l10n_nl_xaf_auditfile_export2/l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py#L42-L49

Reply:
Looks like you have it all set up in your branch. Care to make a PR into mine?

Reply:
Thanks, merged!